tutorials/hosted_phoenix/hosted_phoenix_openai_tutorial.ipynb

```diff
@@ -38,7 +38,7 @@
    "metadata": {},
```

Comment:
<p>Same as others, this is more install than is needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>